docs-source/content/userguide/managing-domains/model-in-image/overview.md

```diff
@@ -0,0 +1,59 @@
++++
+title = "Overview"
+date = 2020-03-11T16:45:16-05:00
+weight = 10
+pre = "<b> </b>"
+description = "Introduction to Model in Image, description of its runtime behavior, and references."
++++
+
+#### Content
+
+ - [Introduction](#introduction)
+ - [Runtime behavior overview](#runtime-behavior-overview)
+ - [Runtime updates overview](#runtime-updates-overview)
+ - [References](#references)
+
+#### Introduction
+
+Model in Image is an alternative to the operator's Domain in Image and Domain on PV domain types. See [Choose a domain home source type]({{< relref "/userguide/managing-domains/choosing-a-model/_index.md" >}}) for a comparison of operator domain types.
+
+Unlike Domain in PV and Domain in Image, Model in Image eliminates the need to pre-create your WebLogic domain home prior to deploying your domain resource.
+
+It enables:
+
+ - Defining a WebLogic domain home configuration using WebLogic Deploy Tool (WDT) model files and application archives.
+ - Embedding model files and archives in a custom Docker image, and using the WebLogic Image Tool (WIT) to generate this image.
+ - Supplying additional model files using a Kubernetes config map.
+ - Supplying Kubernetes secrets that resolve macro references within the models. For example, a secret can be used to supply a database credential.
+ - Updating WDT model files at runtime. For example, you can add a data source to a running domain. Note that all such updates currently cause the domain to 'roll' in order to take effect.
+ - Deploying standard WLS domains, Restricted JRF domains, or JRF domains.
+
+WDT models are a convenient and simple alternative to WebLogic WLST configuration scripts and templates. They compactly define a WebLogic domain using YAML files and support including application archives in a ZIP file.  The WDT model format is described in the open source [WebLogic Deploy Tool](https://github.com/oracle/weblogic-deploy-tooling) GitHub project.
+
+For JRF domains, Model in Image provides additional support for (a) initializing the RCU database for a domain when a domain is started for first time, (b) supplying an RCU password, and finally (c) obtaining an RCU wallet for re-use in subsequent restarts of the same domain. See [Prerequisites for JRF domain types]({{< relref "/userguide/managing-domains/model-in-image/usage/_index.md#7-prerequisites-for-jrf-domain-types" >}}) and [Reusing an RCU database]({{< relref "/userguide/managing-domains/model-in-image/reusing-rcu.md" >}}).
```

Comment:
@rosemarymarano, this list using (a), (b), (c) gets parsed strangely in markdown because (c) becomes the copyright symbol.  How should we organize this?

Reply:
It looks ok in my browser (Firefox). But, if it is a problem in other browsers we can simply remove the letters. They are not needed.

Reply:
It looks fine for me too on Chrome.  @markxnelson, does this look better now that you've updated Hugo?